groundskeeper

```diff
@@ -10,6 +10,10 @@ STUDIO_TYPE="groundskeeper"
 DEBUG=${DEBUG:-0}
 ALL_PLANS=()
 SKIP_CLONE=${SKIP_CLONE:-0}
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd )"
+RELEASE_MONITORING_ID_FILE="${SCRIPT_DIR}/release_monitoring_ids"
+
+declare -A release_monitoring_ids
```

Comment:
The use of associative arrays means we now require Bash 4.0+

Reply:
I'm OK with that :)